src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-block: 32px 16px;
+  padding-inline: 16px;
+
+  &__image {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+      background-image: url(/src/images/imac.jpeg);
+      background-size: 100% 100%;
+    }
+  }
+
+  &__description {
+    margin-block: 40px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-text-color;
+  }
+
+  &__product-code {
+    margin-block: 4px 16px;
```

Comment:
Similarly to the previous comment on `margin-block`, use `margin-top` and `margin-bottom` instead for better browser support.